website_hr_contact/controllers/main.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import werkzeug
+
+from odoo import http
+from odoo.addons.website.models.website import slug
+from odoo.osv.orm import browse_record
+
+
+class QueryURL(object):
```

Comment:
any reason not to use `from odoo.addons.website.controllers.main import QueryURL`?